src/views/ApartmentApp/components/StudentApplication/index.js

```diff
@@ -657,16 +748,26 @@ const StudentApplication = ({ userProfile, authentication }) => {
               <Grid container direction="row" justify="center" spacing={2}>
                 <Grid container item xs={12} md={8} lg={6} direction="column" spacing={2}>
                   <Grid item>
-                    <HallSelection
-                      authentication
-                      editorUsername={editorUsername}
-                      preferredHalls={preferredHalls}
-                      saving={saving}
-                      onHallAdd={handleHallAdd}
-                      onHallInputChange={handleHallInputChange}
-                      onHallRemove={handleHallRemove}
-                      onSaveButtonClick={handleSaveButtonClick}
-                    />
+                    {userProfile.AD_Username === editorUsername ? (
```

Comment:
Okay so for non-editors we are making it that they can't see the application then. Got it.